src/app/components/home/home.component.spec.ts

```diff
@@ -435,12 +435,14 @@ describe('HomeComponent', () => {
     });
 
     it('should skip creating a chart if one already exists', () => {
+      spyOn(component, 'updateData').and.callFake(function(){});
```

Comment:
what does this do?

Reply:
This prevents the updateData call from being executed as it was erroring out and it is not relevant to the test